src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/swagger-summary-description.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
just curios, could we leverage already existing test file that we are using for other description checks that way we do not explode test files with rules.